server/api/src/main/java/org/hawkular/apm/server/api/model/zipkin/Span.java

```diff
@@ -225,4 +231,27 @@ public String componentType() {
         return null;
     }
 
+    public List<Property> properties() {
+        List<Property> properties = new ArrayList<>();
+
+        if (binaryAnnotations != null) {
+            Set<String> services = new HashSet<>();
+
+            for (BinaryAnnotation binaryAnnotation : binaryAnnotations) {
+                Property property = new Property(binaryAnnotation.getKey(), binaryAnnotation.getValue());
```

Comment:
We need to provide the property type. There should be a mapping from https://github.com/openzipkin/brave/blob/master/brave-core/src/main/java/com/twitter/zipkin/gen/AnnotationType.java to the PropertyType enum values. Currently don't need to be as detailed - so many will just be listed as numeric - and should create additional PropertyType values for Boolean and Binary? Default should be String.